### DISCUSSION TITLE
求梅林 非Koolshare版本 的安装包

### DISCUSSION THREAD

**Discussion by johnnyz128** [messense/aliyundrive-webdav#330] [Q&A] [answered]
安装报错：./install.sh: source: line 2: can't open '/koolshare/scripts/base.sh'
我用的是从 https://www.asuswrt-merlin.net/ 下载的Merlin固件
请问有非Koolshare版本 的梅林安装包吗？
谢谢！

**Comment by messense** [comment, marked as answer]
可以直接下载对应 CPU 架构的 tar.gz 包命令行运行：https://github.com/messense/aliyundrive-webdav/releases/tag/v1.2.6

**Reply by johnnyz128** [reply]
感谢回复！
# opkg print-architecture 
arch all 100
arch aarch64-3.10 160

# opkg install aliyundrive-webdav_1.2.6-1_aarch64_generic.ipk 
Unknown package 'aliyundrive-webdav'.
Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures conf
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.

修改 /opt/etc/opkg.conf 增加：
arch aarch64_generic 100

# opkg install aliyundrive-webdav_1.2.6-1_aarch64_generic.ipk
Installing aliyundrive-webdav (1.2.6-1) to root...
Configuring aliyundrive-webdav.
Collected errors:
 * wfopen: /etc/init.d/aliyundrive-webdav: Read-only file system.
 * wfopen: /usr/bin/aliyundrive-webdav: Read-only file system.
 
使用 opkg install -d 指定安装路径 似乎解决了这些问题。
